README.md

```diff
@@ -1,9 +1,17 @@
-![Cicero](./cicero.png)
+<p align="center">
+  <a href="./cicero.png">
+    <img src="./cicero.png" alt="Cicero logo">
+  </a>
+</p>
+
 
 [![Build Status](https://travis-ci.org/accordproject/cicero.svg?branch=master)](https://travis-ci.org/accordproject/cicero)
 [![Coverage Status](https://coveralls.io/repos/github/accordproject/cicero/badge.svg?branch=master)](https://coveralls.io/github/accordproject/cicero?branch=master)
+[![GitHub license](https://img.shields.io/github/license/accordproject/ergo?color=bright-green)](./LICENSE)
 [![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 [![npm version](https://badge.fury.io/js/%40accordproject%2Fcicero-cli.svg)](https://badge.fury.io/js/%40accordproject%2Fcicero-cli)
+[![join slack](https://img.shields.io/badge/Slack-Join%20Slack-blue)](https://accord-project-slack-signup.herokuapp.com/)
+
```

Comment:
Are you able to center these as well, to fit under `Cicero`'s logo?

Reply:
Done. Please review!